src/data/real/hyperreal.lean

```diff
@@ -621,12 +622,12 @@ infinitesimal_sub_is_st $ is_st_st' hx
 lemma infinite_pos_iff_infinitesimal_inv_pos {x : ℝ*} :
   infinite_pos x ↔ (infinitesimal x⁻¹ ∧ x⁻¹ > 0) :=
 ⟨ λ hip, ⟨ infinitesimal_def.mpr $ λ r hr,
-  ⟨ lt_trans (of_lt_of_lt U (neg_neg_of_pos hr)) (inv_pos.2 (hip 0)),
-    (inv_lt (of_lt_of_lt U hr) (hip 0)).mp (by convert hip r⁻¹) ⟩,
+  ⟨ lt_trans (coe_lt_coe.2 (neg_neg_of_pos hr)) (inv_pos.2 (hip 0)),
+    (inv_lt (coe_lt_coe.2 hr) (hip 0)).mp (by convert hip r⁻¹) ⟩,
```

Comment:
I really don't want Yury to spend time on this exotic file, but `norm_cast` should be able to simplify all this.

Reply:
I tried to minimize modifications in this file. It is possible to rewrite many proofs here using `norm_cast` but I don't want to do this.